packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/PaymentMethods/selectors.ts

```diff
@@ -0,0 +1,16 @@
+import { lift } from 'ramda'
+import { selectors } from 'data'
+
+export const getData = state => {
+  const cardsR = selectors.components.simpleBuy.getSBCards(state)
+  const eligibilityR = selectors.components.simpleBuy.getSBFiatEligible(state)
+  const paymentMethodsR = selectors.components.simpleBuy.getSBPaymentMethods(
+    state
+  )
+
+  return lift((cards, eligibility, paymentMethods) => ({
+    cards,
+    eligibility,
+    paymentMethods
+  }))(cardsR, eligibilityR, paymentMethodsR)
+}
```

Comment:
Please use new ExtractSuccess type so we can remove https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/2521/files#diff-723ce43ac4609d54ddb1dcf959a63007R55-R57. I will write a wiki article on this manana and update the snippets!

Reply:
@plondon didn't get you about this one

Reply:
Check out this example: https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/feat/sell-p1/packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CryptoSelection/selectors.ts/#L1-L22

Which allows you to not manually type out [this](https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/2521/files#diff-723ce43ac4609d54ddb1dcf959a63007R54-R58)

export type SuccessStateType = ReturnType<typeof getData>['data']